tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -532,6 +532,11 @@
   ansible.builtin.include_tasks:
     file: neutron_verify.yaml
 
+- name: Adopt Octavia Certificates Wallaby->Antelope FFU
```

Comment:
these are passing in the last zuul run here for the standalone 


        * https://logserver.rdoproject.org/56/556/e29930087fc1e324212e497e62c74e3b12337c1a/github-check/adoption-standalone-to-crc-ceph/4538bfa/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2024-10-08T08:41:48EDT.log
        * TASK [dataplane_adoption : Adopt Octavia Certificates Wallaby->Antelope FFU] ***
included: /home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption/tests/roles/dataplane_adoption/tasks/octavia_certs.yaml for localhost